### ISSUE TITLE
Set the model to evaluation model in notebook

### ISSUE THREAD

**Issue by melodiemonod** [Novartis/torchsurv#64]
The model was not set to evaluation mode in the notebook before evaluating the parameters on test. We included dropout layers. They will randomly set a fraction of the inputs to zero during training, which helps prevent overfitting. However, dropout is also active during evaluation if the model is not set to evaluation mode.

I added ```weibull_model.eval()``` and ```cox_model.eval()``` before the parameters evaluation on data from ```dataloader_test```.
